cstruct.opam

```diff
@@ -12,7 +12,7 @@ build: [
   ["jbuilder" "subst" "-p" name "--name" name] {pinned}
   ["jbuilder" "build" "-p" name "-j" jobs]
 ]
-build-depends: ["jbuilder" "runtest" "-p" name]
+build-test: ["jbuilder" "runtest" "-p" name]
```

Comment:
I suspect you can directly push to this branch if you want to fix it.  I feel like I already fixed enough regressions today (within this PR, just look at the commit messages).